distribution/docs/src/main/resources/_contents/_endpoints/opensearch-endpoint-contents.adoc

```diff
@@ -0,0 +1,276 @@
+|`lon`
+|`EPSG:4326` decimal degrees
+|Expects a longitude and a radius to be specified.
+
+|`radius`
+|`radius`
+|Meters along the Earth's surface > 0
+|Used in conjunction with lat and lon query parameters.
+
+|`polygon`
+|`polygon`
+|clockwise `lat lon` pairs ending at the first one
+|example: `-80, -170, 0, -170, 80, -170, 80, 170, 0, 170, -80, 170, -80, -170`
+
+According to the OpenSearch Geo Specification this is *deprecated*. 
+Use geometry instead.
+
+|`box`
+|`bbox`
+|4 comma-separated `EPSG:4326` decimal degrees
+|west, south, east, north
+
+|`geometry`
+|`geometry` 
+|WKT Geometries: `POINT`, `POLYGON`, `MULTIPOINT`, `MULTIPOLYGON`
+|Examples:
+
+`POINT(10 20)` where 10 is the longitude and 20 is the latitude.
+
+`POLYGON ( ( 30 10, 10 20, 20 40, 40 40, 30 10 ) )`. 30 is longitude and 10 is latitude for the first point.
+Make sure to repeat the starting point as the last point to close the polygon.
+
+|===
+
+.Extensions
+[cols="4*", options="header"]
+|===
+|OS Element
+|HTTP Parameter
+|Possible Values
+|Comments
+
+|`sort`
+|`sort`
+|`sbfield`: 'date' or 'relevance'
+`sborder`: 'asc' or 'desc'
+|`sort=<sbfield>:<sborder>` default: `relevance:desc`
+
+Sorting by date will sort the effective date.
+
+|`maxResults`
+|`mr`
+|Integer >= 0
+|Maximum # of results to return.
+
+If count is also specified, the count value will take precedence over the `maxResults` value
+
+|`maxTimeout`
+|`mt`
+|Integer > 0
+|Maximum timeout (milliseconds) for query to respond
+
+default: 300000 (5 minutes)
+|===
+
+.Federated Search
+[cols="4*", options="header"]
+|===
+|OS Element
+|HTTP Parameter
+|Possible Values
+|Comments
+
+|`routeTo`
+|`src`
+|(varies depending on the names of the sites in the federation)
+|comma delimited list of site names to query.
+
+Also can specify `src=local` to query the local site.
+
+If src is not provided, the default behavior is to execute an enterprise search to the entire federation.
+
+|===
+
+.${branding} Extensions
+[cols="4*", options="header"]
+|===
+|OS Element
+|HTTP Parameter
+|Possible Values
+|Comments
+
+|`dateOffset`
+|`dtoffset`
+|integer > 0
+|Specifies an offset, backwards from the current time, to search on the modified time field for entries. Defined in milliseconds.
+
+|`type`
+|`type`
+|nitf
+|Specifies the type of data to search for.
+
+|`version`
+|`version`
+|20,30
+|Comma-delimited list of version values to search for.
+
+|`selector`
+|`selector`
+|`//namespace:example`,`//example`
+|Comma-delimited list of XPath string selectors that narrow down the search.
+
+|===
+
+====== Supported Complex Contextual Query Format
+
+The OpenSearch Endpoint supports the following operators: `AND`, `OR`, and `NOT`.
+These operators are case sensitive.
+Implicit `ANDs` are also supported.
+
+Using parentheses to change the order of operations is supported.
+Using quotes to group keywords into literal expressions is supported.
+
+The following `EBNF` describes the grammar used for the contextual query format.
```

Comment:
Should we refer (link) to the OpenSearch specification instead of providing the EBNF here? It's pretty difficult to read in PDF format and we shouldn't have to maintain this here anyways.

Should we provide a few examples instead?